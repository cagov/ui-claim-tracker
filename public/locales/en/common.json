{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "EDD Employment Development Department State of California Home",
    "edd-home": "EDD Home",
    "help": "Help",
    "welcome": "Welcome, name",
    "logout": "Log out",
    "uio-home": "UI Online Home"
  },
  "title": "Claim Tracker",
  "welcome": "Claim Tracker",
  "change-locale": "En español",
  "claim-status": {
    "title": "Claim Status",
    "pending-determination": "We need to confirm your eligibility for benefits.",
    "generic-pending": "Your payments are pending further review.",
    "generic-all-clear": "There are no pending issues at this time.",
    "next-steps": "Next Steps"
  },
  "claim-details": {
    "title": "Claim Details",
    "benefit-year": "Benefit Year",
    "program-type": "Program Type",
    "claim-balance": "Claim Balance",
    "weekly-benefit-amount": "Weekly Benefit Amount",
    "last-payment-issued": "Last Payment Issued",
    "extension-type": "Extension Type",
    "extension-end-date": "Extension End Date"
  },
  "work-in-progress": {
    "title": "This page is a work in progress.",
    "message": "While we test this new part of UI Online, the information here may be limited for your claim. We're working to add information about common issues that lead to <1>pending payments</1>."
  },
  "footer": {
    "toTop": "Back to Top",
    "contact": "Contact EDD",
    "conditionsOfUse": "Conditions of Use",
    "privacyPolicy": "Privacy Policy",
    "accessibility": "Accessibility",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "Your Session Will End Soon",
    "warning": "To protect your information, you will be logged out in {{numberOfMinutes}} minutes if you do not continue.",
    "button": "Stay Logged In"
  }
}
