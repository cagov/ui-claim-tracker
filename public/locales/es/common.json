{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "Página principal del Departamento de Desarrollo de Empleo del EDD",
    "edd-home": "EDD Casa",
    "help": "Ayuda",
    "welcome": "Bienvenidas, nombre",
    "logout": "Cerrar sesión",
    "uio-home": "UI Online Casa"
  },
  "title": "Claim Tracker",
  "welcome": "Claim Tracker",
  "change-locale": "In English",
  "claim-status": {
    "title": "Claim Status",
    "next-steps": "Próximos Pasos"
  },
  "claim-details": {
    "title": "Claim Details",
    "benefit-year": "Año de beneficio",
    "program-type": "Tipo de Programa",
    "claim-balance": "Balance de Reclamación",
    "weekly-benefit-amount": "Monto de Beneficio Semanal",
    "last-payment-issued": "Último Pago Emitido",
    "extension-type": "Tipo de Extensión",
    "extension-end-date": "Fecha de Finalización de la Extensión"
  },
  "work-in-progress": {
    "title": "Esta página es un trabajo en progreso.",
    "message": "Mientras probamos esta nueva parte de UI Online, la información aquí puede ser limitada para su reclamo. Estamos trabajando para agregar información sobre problemas comunes que generan <1>pagos pendientes</1>."
  },
  "footer": {
    "toTop": "Volver Arriba",
    "contact": "Contacta con EDD",
    "conditionsOfUse": "Condiciones de Uso",
    "privacyPolicy": "Política de Privacidad",
    "accessibility": "Accesibilidad",
    "copyright": "Copyright © 2021 Estado de California"
  },
  "timeout-modal": {
    "header": "Tu sesión terminará pronto",
    "warning": "Para proteger su información, se cerrará la sesión en {{numberOfMinutes}} minutos si no continúa.",
    "button": "Sigue Trabajando"
  }
}
