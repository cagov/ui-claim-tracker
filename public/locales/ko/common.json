{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "EDD 캘리포니아주 고용개발부 홈",
    "edd-home": "EDD 홈",
    "help": "도움말",
    "logout": "로그아웃",
    "uio-home": "실업 보험 홈",
    "uio-certify": "인증",
    "uio-payments": "지급",
    "uio-history": "기록",
    "uio-status": "상태",
    "uio-profile": "프로필",
    "uio-inbox": "메시지함",
    "uio-contact": "문의"
  },
  "title": "청구 상태 추적기",
  "welcome": "청구 상태 추적기",
  "errors": {
    "not-found": {
      "label": "해당 페이지를 찾을 수 없습니다.",
      "entry": "귀하가 클릭한 링크가 손상되었거나 페이지가 제거되었을 수 있습니다."
    },
    "button": "실업 보험 홈으로 돌아가기",
    "server-error": {
      "label": "EDD에서 오류가 발생했습니다.",
      "entry": "귀하의 요청을 처리할 수 없습니다."
    }
  },
  "maintenance": {
    "label": "곧 서비스를 재개하겠습니다",
    "entry": "현재 시스템 점검 중이어서 이 페이지를 사용할 수 없습니다.",
    "entry2": "서비스를 개선하는 동안 양해해 주셔서 감사합니다."
  },
  "claim-status": {
    "title": "청구 상태",
    "your-next-steps": "다음 단계",
    "edd-next-steps": "EDD 다음 단계"
  },
  "claim-details": {
    "title": "청구 상세 정보",
    "benefit-year": "급여 연도",
    "program-type": "청구 유형",
    "claim-balance": "청구 잔액",
    "weekly-benefit-amount": "주간 급여 액수",
    "last-payment-issued": "지급된 마지막 급여",
    "none": "없음",
    "extension-type": "연장 유형"
  },
  "footer": {
    "toTop": "위로 올라가기",
    "contact": "EDD에 문의하기",
    "conditionsOfUse": "사용 조건",
    "privacyPolicy": "개인정보 보호정책",
    "accessibility": "접근성",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "세션이 곧 종료됩니다",
    "warning": "활동이 없으면 정보를 보호하기 위해 {{count}}분 후에 로그아웃됩니다.",
    "warning_plural": "활동이 없으면 정보를 보호하기 위해 {{count}}분 후에 로그아웃됩니다.",
    "button": "실업 보험 홈으로 돌아가기"
  },
  "time": {
    "between-range": ", {{ampm}} {{range}}시(태평양 시간)",
    "between-start-end": ", {{start.ampm}} {{start.time}}시 ~ {{end.ampm}} {{end.time}}시(태평양 시간)",
    "am": "오전",
    "pm": "오후"
  },
  "urls": {
    "edd": {
      "ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
      "ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
      "ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm",
      "ui-calculator": "https://edd.ca.gov/Unemployment/UI-Calculator.htm"
    },
    "hypertext": {
      "uio-homepage": "실업 보험 온라인 홈페이지",
      "pending-payment": "청구 상태: 보류 중인 지급",
      "certify": "급여에 대한 인증",
      "reopen": "청구 재개",
      "reapply": "실업 급여 재신청",
      "enough-wages": "충분한 임금"
    }
  },
  "external-navigation-modal": {
    "header": "탐색 확인",
    "warning": "애플리케이션 밖에서 새 탭을 열고 있습니다. 계속을 클릭하여 새 탭을 열거나 취소를 선택하여 애플리케이션으로 돌아가세요.",
    "cancel-button": "취소",
    "continue-button": "계속"
  }
}
