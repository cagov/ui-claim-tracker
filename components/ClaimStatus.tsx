import { useTranslation } from 'next-i18next'

import { NextSteps } from './NextSteps'
import { TextLine } from './TextLine'
import { TransLine } from './TransLine'
import { ClaimStatusContent } from '../types/common'

export interface ClaimStatusProps extends ClaimStatusContent {
  loading: boolean
  userArrivedFromUioMobile: boolean
}

export const ClaimStatus: React.FC<ClaimStatusProps> = ({
  loading = false,
  userArrivedFromUioMobile = false,
  heading,
  summary,
  yourNextSteps,
  eddNextSteps,
}) => {
  const { t } = useTranslation(['common', 'claim-status'])

  return (
    <div className="claim-status claim-section">
      <h2>{t('claim-status.title')}</h2>
      <div className="pending-status claim-subsection">
        <TextLine loading={loading} header text={t(heading)} />
      </div>
      <div className="summary">
        <TransLine
          loading={loading}
          userArrivedFromUioMobile={userArrivedFromUioMobile}
          i18nKey={summary.i18nKey}
          links={summary.links}
        />
      </div>
      <NextSteps
        loading={loading}
        userArrivedFromUioMobile={userArrivedFromUioMobile}
        header={t('claim-status.your-next-steps')}
        nextSteps={yourNextSteps}
      />
      <NextSteps
        loading={loading}
        userArrivedFromUioMobile={userArrivedFromUioMobile}
        header={t('claim-status.edd-next-steps')}
        nextSteps={eddNextSteps}
      />
    </div>
  )
}
