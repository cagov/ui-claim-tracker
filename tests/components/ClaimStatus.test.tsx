import renderer from 'react-test-renderer'
import { ClaimStatus } from '../../components/ClaimStatus'
import getScenarioContent, { ScenarioType } from '../../utils/getScenarioContent'
import apiGatewayStub from '../../utils/apiGatewayStub'
import { ClaimStatusContent } from '../../types/common'

function renderClaimStatusComponent(statusContent: ClaimStatusContent, userArrivedFromUioMobile: boolean): string {
  return renderer
    .create(
      <ClaimStatus
        loading={false}
        userArrivedFromUioMobile={userArrivedFromUioMobile}
        heading={statusContent.heading}
        summary={statusContent.summary}
        yourNextSteps={statusContent.yourNextSteps}
        eddNextSteps={statusContent.eddNextSteps}
      />,
    )
    .toJSON()
}

function testClaimStatus(
  scenarioType: ScenarioType,
  hasCertificationWeeksAvailable: boolean,
  userArrivedFromUioMobile: boolean,
): string {
  const scenarioContent = getScenarioContent(apiGatewayStub(scenarioType, hasCertificationWeeksAvailable))
  return renderClaimStatusComponent(scenarioContent.statusContent, userArrivedFromUioMobile)
}

describe('Scenario 1', () => {
  it('matches when there are weeks to certify, on desktop', () => {
    expect(testClaimStatus(ScenarioType.Scenario1, true)).toMatchSnapshot()
  })
  it('matches when there are weeks to certify, on mobile', () => {
    expect(testClaimStatus(ScenarioType.Scenario1, true, true)).toMatchSnapshot()
  })

  it("matches when there aren't weeks to certify, on desktop", () => {
    expect(testClaimStatus(ScenarioType.Scenario1, false)).toMatchSnapshot()
  })
  it("matches when there aren't weeks to certify, on mobile", () => {
    expect(testClaimStatus(ScenarioType.Scenario1, false, true)).toMatchSnapshot()
  })
})

describe('Scenario 4', () => {
  it('matches when there are weeks to certify, on desktop', () => {
    expect(testClaimStatus(ScenarioType.Scenario4, true)).toMatchSnapshot()
  })
  it('matches when there are weeks to certify, on mobile', () => {
    expect(testClaimStatus(ScenarioType.Scenario4, true, true)).toMatchSnapshot()
  })

  it("matches when there aren't weeks to certify, on desktop", () => {
    expect(testClaimStatus(ScenarioType.Scenario4, false)).toMatchSnapshot()
  })
  it("matches when there aren't weeks to certify, on mobile", () => {
    expect(testClaimStatus(ScenarioType.Scenario4, false, true)).toMatchSnapshot()
  })
})

// Note that Scenarios 5 & 6 explicitly differ based on whether hasCertificationWeeksAvailable
// is true or false, so it doesn't make sense to test again.
describe('Scenario 5', () => {
  it('matches, on desktop', () => {
    expect(testClaimStatus(ScenarioType.Scenario5)).toMatchSnapshot()
  })
  it('matches, on mobile', () => {
    expect(testClaimStatus(ScenarioType.Scenario5, true, true)).toMatchSnapshot()
  })
})

describe('Scenario 6', () => {
  it('matches, on desktop', () => {
    expect(testClaimStatus(ScenarioType.Scenario6)).toMatchSnapshot()
  })
  it('matches, on mobile', () => {
    expect(testClaimStatus(ScenarioType.Scenario6, true, true)).toMatchSnapshot()
  })
})
