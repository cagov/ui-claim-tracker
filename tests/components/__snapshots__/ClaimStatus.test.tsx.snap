// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scenario 1 matches when there are weeks to certify, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Pending Eligibility — Phone Interview Will Be Scheduled
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue that could make you ineligible for benefits. We need to confirm your eligibility in a phone interview.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Continue to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
           while we determine your eligibility.
        </li>
        <li
          className="next-step"
        >
          Confirm we have your current phone number and mailing address. Go to the 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
          , select 
          <strong>
            Profile
          </strong>
           from the main menu, then select 
          <strong>
            Contact Information
          </strong>
          .
        </li>
        <li
          className="next-step"
        >
          Check your mail for the 
          &lt;em&gt;Notification of Unemployment Insurance Benefits Eligibility Interview&lt;/em&gt;
           (DE 4800). It will explain the issue and help you prepare for the interview.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will schedule a phone interview to determine your eligibility for benefits. This page will be updated with appointment details once your interview is scheduled.
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 1 matches when there are weeks to certify, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Pending Eligibility — Phone Interview Will Be Scheduled
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue that could make you ineligible for benefits. We need to confirm your eligibility in a phone interview.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Continue to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
           while we determine your eligibility.
        </li>
        <li
          className="next-step"
        >
          Confirm we have your current phone number and mailing address. Go to the 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
          , select 
          <strong>
            Profile
          </strong>
           from the main menu, then select 
          <strong>
            Contact Information
          </strong>
          .
        </li>
        <li
          className="next-step"
        >
          Check your mail for the 
          &lt;em&gt;Notification of Unemployment Insurance Benefits Eligibility Interview&lt;/em&gt;
           (DE 4800). It will explain the issue and help you prepare for the interview.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will schedule a phone interview to determine your eligibility for benefits. This page will be updated with appointment details once your interview is scheduled.
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 1 matches when there aren't weeks to certify, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Pending Eligibility — Phone Interview Will Be Scheduled
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue that could make you ineligible for benefits. We need to confirm your eligibility in a phone interview.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Confirm we have your current phone number and mailing address. Go to the 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
          , select 
          <strong>
            Profile
          </strong>
           from the main menu, then select 
          <strong>
            Contact Information
          </strong>
          .
        </li>
        <li
          className="next-step"
        >
          Check your mail for the 
          &lt;em&gt;Notification of Unemployment Insurance Benefits Eligibility Interview&lt;/em&gt;
           (DE 4800). It will explain the issue and help you prepare for the interview.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will schedule a phone interview to determine your eligibility for benefits. This page will be updated with appointment details once your interview is scheduled.
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 1 matches when there aren't weeks to certify, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Pending Eligibility — Phone Interview Will Be Scheduled
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue that could make you ineligible for benefits. We need to confirm your eligibility in a phone interview.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Confirm we have your current phone number and mailing address. Go to the 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
          , select 
          <strong>
            Profile
          </strong>
           from the main menu, then select 
          <strong>
            Contact Information
          </strong>
          .
        </li>
        <li
          className="next-step"
        >
          Check your mail for the 
          &lt;em&gt;Notification of Unemployment Insurance Benefits Eligibility Interview&lt;/em&gt;
           (DE 4800). It will explain the issue and help you prepare for the interview.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will schedule a phone interview to determine your eligibility for benefits. This page will be updated with appointment details once your interview is scheduled.
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 4 matches when there are weeks to certify, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Review Required
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue and may need to determine your eligibility or verify your information.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Continue to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
           while we determine your eligibility.
        </li>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          For more information, visit 
          <a
            href="https://www.edd.ca.gov/unemployment/claim-status.htm"
          >
            Claim Status: Pending Payment
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will contact you if we need more information to resolve the issue.
        </li>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 4 matches when there are weeks to certify, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Review Required
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue and may need to determine your eligibility or verify your information.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Continue to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
           while we determine your eligibility.
        </li>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          For more information, visit 
          <a
            href="https://www.edd.ca.gov/unemployment/claim-status.htm"
          >
            Claim Status: Pending Payment
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will contact you if we need more information to resolve the issue.
        </li>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 4 matches when there aren't weeks to certify, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Review Required
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue and may need to determine your eligibility or verify your information.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          For more information, visit 
          <a
            href="https://www.edd.ca.gov/unemployment/claim-status.htm"
          >
            Claim Status: Pending Payment
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will contact you if we need more information to resolve the issue.
        </li>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 4 matches when there aren't weeks to certify, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Review Required
    </h3>
  </div>
  <div
    className="summary"
  >
    We identified an issue and may need to determine your eligibility or verify your information.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          For more information, visit 
          <a
            href="https://www.edd.ca.gov/unemployment/claim-status.htm"
          >
            Claim Status: Pending Payment
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will contact you if we need more information to resolve the issue.
        </li>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 5 matches, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      No Weeks Available to Certify
    </h3>
  </div>
  <div
    className="summary"
  >
    You currently have no weeks available to certify for benefits.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          If it’s been more than 30 days since you last certified for benefits, you must 
          <a
            href="https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm"
          >
            reopen your claim
          </a>
           before you can certify.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 5 matches, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      No Weeks Available to Certify
    </h3>
  </div>
  <div
    className="summary"
  >
    You currently have no weeks available to certify for benefits.
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Check your 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and inbox for the latest updates.
        </li>
        <li
          className="next-step"
        >
          Check your mail. We may send requests for additional information. Make sure to respond as requested to avoid delays in processing your claim.
        </li>
        <li
          className="next-step"
        >
          If it’s been more than 30 days since you last certified for benefits, you must 
          <a
            href="https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm"
          >
            reopen your claim
          </a>
           before you can certify.
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          We will email you when you have weeks available to 
          <a
            href="https://edd.ca.gov/Unemployment/certify.htm"
          >
            certify for benefits
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 6 matches, on desktop 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Weeks Available to Certify
    </h3>
  </div>
  <div
    className="summary"
  >
    You have weeks available to 
    <a
      href="https://edd.ca.gov/Unemployment/certify.htm"
    >
      certify for benefits
    </a>
    .
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Select 
          <strong>
            Certify for Benefits
          </strong>
           in the Notification section on your 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          When we process your certification, we will update the payment status. Go to the 
          <a
            href="https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and select 
          <strong>
            History
          </strong>
           from the main menu.
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Scenario 6 matches, on mobile 1`] = `
<div
  className="claim-status claim-section"
>
  <h2>
    Claim Status
  </h2>
  <div
    className="pending-status claim-subsection"
  >
    <h3
      className="header-line"
    >
      Weeks Available to Certify
    </h3>
  </div>
  <div
    className="summary"
  >
    You have weeks available to 
    <a
      href="https://edd.ca.gov/Unemployment/certify.htm"
    >
      certify for benefits
    </a>
    .
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      Your Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          Select 
          <strong>
            Certify for Benefits
          </strong>
           in the Notification section on your 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
          .
        </li>
      </ul>
    </div>
  </div>
  <div
    className="next-steps claim-subsection"
  >
    <h3
      className="next-steps-header"
    >
      EDD Next Steps
    </h3>
    <div
      className="next-step-explanation"
    >
      <ul>
        <li
          className="next-step"
        >
          When we process your certification, we will update the payment status. Go to the 
          <a
            href="https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx"
          >
            UI Online homepage
          </a>
           and select 
          <strong>
            History
          </strong>
           from the main menu.
        </li>
      </ul>
    </div>
  </div>
</div>
`;
